### PR TITLE
AP-4132 combine property pages

### PR DESCRIPTION
**Pull request by RoseSAK**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4132)

Combined the four property pages `property_value`, `outstanding_mortgage`, `shared_ownership` and `percentage_homes` into a single page `property_details`
Removed all controllers, routes, views, forms and specs for the old pages

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: spec/requests/providers/offline_accounts_spec.rb [hunk 1]
```diff
@@ -38,21 +38,12 @@
           end
         end
 
-        context "when applicant owns home with shared ownership" do
-          before { get providers_legal_aid_application_means_percentage_home_path(application) }
+        context "when applicant owns home" do
+          before { get providers_legal_aid_application_means_property_details_path(application) }
 
           it "points to percentage owned page" do
```
Comment (colinbruce) [inline spec/requests/providers/offline_accounts_spec.rb:44]: One *tiny* nitpick... the test description should change to property details

File: app/helpers/check_answer_url_helper.rb [hunk 2]
```diff
@@ -6,16 +6,6 @@ def check_answer_url_for(journey_type, field_name, application = nil, args = {})
       legal_aid_application: application,
       params: args,
     )
-    anchor = field_name_to_anchor_map[field_name]
```
Comment (colinbruce) [inline app/helpers/check_answer_url_helper.rb:9]: This is weird... it implies this was once a single page where the CYA could link to a specific field 🤔 
Are we reverting to what was, and should we be linking to a specific field again?

**Review by colinbruce — APPROVED** [review]
Awesome work! 🏆 
One minor nitpick and one observation, happy to discuss

File: spec/requests/providers/offline_accounts_spec.rb (hunk 1 above)
Comment (RoseSAK) [reply spec/requests/providers/offline_accounts_spec.rb:44]: Ah yes good spot, thank you :)

**Pull request merged**